stor/dx.py

```diff
@@ -967,13 +1006,30 @@ def listdir(self, only='all', canonicalize=False):
 
     def listdir_iter(self, canonicalize=False):
         """Iterate the path as a dir, returning top-level directories and files.
+        If listing an empty path, lists the projects a user has access to.
```

Comment:
needs more context (especially in docs) about which projects are seen when you do ls on `dx://`

Reply:
Added